src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -624,11 +624,11 @@ instruct vsubL_vx(vReg dst, vReg src1, iRegL src2) %{
 
 // vector-scalar sub (predicated)
 
-instruct vsubI_vx_masked(vReg dst_src, iRegIorL2I src2, vRegMask_V0 v0) %{
+instruct vsub_vx_masked(vReg dst_src, iRegIorL2I src2, vRegMask_V0 v0) %{
   match(Set dst_src (SubVB (Binary dst_src (Replicate src2)) v0));
   match(Set dst_src (SubVS (Binary dst_src (Replicate src2)) v0));
   match(Set dst_src (SubVI (Binary dst_src (Replicate src2)) v0));
-  format %{ "vsubI_vx_masked $dst_src, $dst_src, $src2, $v0" %}
+  format %{ "vsub_vx_masked $dst_src, $dst_src, $src2, $v0" %}
```

Comment:
Can you update the format of `instruct vsubL_vx_masked` while you are on it?
It should be `vsubL_vx_masked` instead of `vsub_vx_masked`.